include/irr/asset/IAssetLoader.h

```diff
@@ -101,6 +101,9 @@ class IAssetLoader : public virtual core::IReferenceCounted
         const E_CACHING_FLAGS cacheFlags;
         const char* relativeDir;
         const E_LOADER_PARAMETER_FLAGS loaderFlags;				//!< Flags having an impact on extraordinary tasks during loading process
+
+		//smart ptr?
+		CQuantNormalCache& quantNormalCache = IMeshManipulator::getQuantNormalCache();
```

Comment:
@devshgraphicsprogramming is it ok?

Reply:
https://github.com/Przemog1/IrrlichtBAW/blob/shader_pipeline/include/irr/asset/IMeshManipulator.h#L374

Reply:
just plain old pointer and  non-static member function